datalogtool/src/main/native/cpp/Exporter.cpp

```diff
@@ -180,17 +180,18 @@ InputFile::~InputFile() {
 }
 
 static std::unique_ptr<InputFile> LoadDataLog(std::string_view filename) {
-  std::error_code ec;
-  auto buf = wpi::MemoryBuffer::GetFile(filename, ec);
-  std::string fn{filename};
-  if (ec) {
+  std::unique_ptr<wpi::MemoryBuffer> fileBuffer;
```

Comment:
Inverting the conditional makes it a bit cleaner. It's basically the same structure as main, but with the intent expressed better.
```cpp
 static std::unique_ptr<InputFile> LoadDataLog(std::string_view filename) {
  auto buf = wpi::MemoryBuffer::GetFile(filename);
  if (!buf) {
    return std::make_unique<InputFile>(
        filename,
        fmt::format("Could not open file: {}", buf.error().message()));
  }

  wpi::log::DataLogReader reader{std::move(buf.value())};
```